### PR TITLE
scroll-timeline-* properties

### PR DESCRIPTION
**Pull request by hamishwillee**
Added the named scroll timeline properties. Other docs work can be tracked in #21069

As previously, attempting to follow existing patterns.

### CODE REVIEW

File: css/properties.json [hunk 1]
```diff
@@ -8369,6 +8369,63 @@
...
+    "percentages": "no",
+    "groups": [
+      "CSS Animations"
+    ],
+    "initial": "none",
+    "appliesto": "scrollContainers",
+    "computed": "asSpecified",
+    "order": "perGrammar",
+    "status": "experimental",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/scroll-timeline-name"
```
Comment (hamishwillee) [inline css/properties.json:8411]:
```suggestion
    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/scroll-timeline-axis"
```

**Review by teoli2003 — APPROVED** [review]

**Comment by hamishwillee** [comment]
Thanks. Can you merge as well - I don't have any rights in this repo

**Comment by hamishwillee** [comment]
@teoli2003 or @wbamberg Can you please merge this? I have no rights here.

**Comment by hamishwillee** [comment]
Or give me right to merge :-)

**Comment by wbamberg** [comment]
> Or give me right to merge :-)

I can't do that but I can merge.

**Pull request merged**